src/lesson6/taskNumber1/CreditCard.java

```diff
@@ -0,0 +1,51 @@
+package lesson6.taskNumber1;
+
+import java.util.Random;
+
+public class CreditCard {
+    private final int id;
+    private static int nextId = 0;
+    private final String accountNumber;
+    private double accountAmount = 10;
+
+
+
+    public CreditCard(){
+        this.id = ++nextId;
+        Random random = new Random();
+        int one = random.nextInt(8999)+1000;
+        int two = random.nextInt(8999)+1000;
+        int free = random.nextInt(8999)+1000;
+        int four = random.nextInt(8999)+1000;
+        String strNumberOne = String.valueOf(one);
+        String strNumberTwo = String.valueOf(two);
+        String strNumberFree = String.valueOf(free);
+        String strNumberFour = String.valueOf(four);
+        accountNumber = (strNumberOne +" "+ strNumberTwo +" "+ strNumberFree +" "+ strNumberFour);
```

Comment:
How about move it from constructor to different method and create a new constructor with account number?